projects/libpsl/build.sh

```diff
@@ -17,6 +17,7 @@
 
 # avoid iconv() memleak on Ubuntu 16.04 image (breaks test suite)
 export ASAN_OPTIONS=detect_leaks=0
+export LD_LIBRARY_PATH=/usr/local/lib
```

Comment:
@rockdaboot the issue is that docker image is used for building only, not for actual running of the fuzzers. Please take a look at the [Runtime Dependencies](https://github.com/google/oss-fuzz/blob/master/docs/fuzzer_environment.md#runtime-dependencies) section.

Is it possible to statically link the fuzz target against the library?